packages/block-library/src/common.scss

```diff
@@ -8,26 +8,12 @@
 	@include gradient-colors-deprecated();
 }
 
-// Font sizes.
-.has-small-font-size {
-	font-size: 0.8125em;
```

Comment:
@jasmussen I've noticed that in the `common.scss` bundled the classes used `em` while in the `editor.scss` bundled they used `px`. Looked at the history but this seems to be like that since the beggining. Do you happen to remember why that difference by any chance? Is it safe to substitute them with the `theme.json` [values](https://github.com/WordPress/gutenberg/blob/trunk/lib/theme.json#L187) in `px`?